src/dune

```diff
@@ -3,6 +3,6 @@
  (public_name coq-paramcoq.plugin)
  (synopsis "Plugin for generating parametricity statements to perform refinement proofs")
  (flags :standard -rectypes -w -9-27)
- (libraries coq-core.plugins.ltac))
+ (libraries coq-core.plugins.ltac)) ; not sure if ltac dep is real
```

Comment:
Maybe not. I'm not even sure why there is a dune file here since this is compiled with coq_makefile, we could probably just remove it.